v4/install/configuration.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Configuration
+nav_order: 3
+layout: default
+parent: Installation
+grand_parent: v4
+---
+# Configuration
+Since PoracleJS requires several services we need to configure them to work together. Some sections will not apply to the Docker configuration, and they will be identified as such. We will go over a basic configuration to get PoracleJS running. After all changes are made to the file the PoracleJS process will need to be restarted to take effect.
+
+## Firewall
+Opening the firewall should only be required on a select-few environments. The use-case for this would be that PoracleJS is required to be accessible on the internet. This is not required for local installations. If you are opening PoracleJS to the internet ensure you are using a reverse proxy for SSL-encrypted traffic. This has the added benefit of not exposing PoracleJS directly to the web. The other use-case would be if your environments default rule is drop. This is not the case for most installations.
+
+PoracleJS runs on 3030/tcp by default unless a different port is selected. Ensure you open this port if its required.
+
+## Geofence
+The geofence file can be found at `./config/geofence.json`. The geofence help define custom areas. While this is not required it can help allow for multiple notification areas to be defined to pinpoint notifications. The easiest way to configure geofences is through the tool [Fence Editor](http://geo.jasparke.net/).
+ * For each area click on `Create` in the top-left corner and fill out the name. This name cannot contain hypens (`-`) but can contain spaces. 
+ * Fill out the field `Fence Name` and click on `Create` to begin drawing the fence
+ * Click where the vertices should be for the area. Once you are done with the area, ensure you click the starting location to close the fence
+ * Perform these steps for all areas you wish to include. Once you have created your areas click on `Save` and `Click to Download`
+
+The downloaded file will be your geofence file. You can either copy the contents into the file on the server or upload it. If you are uploading it ensure the file is owned by the user running PoracleJS.
+
+## Configuration File
+Both installation methods should have a file located at `./config/local.json`. This file will need to be modified to ensure the services are connected together properly.
+
+### Database
+```plain
+This step can be skipped for Docker installations
+```
+The first configuration we need to make is to the Database. This section can be found under the key `database`. Make the following changes to this section:
+ * client
+    * mysql - For use with MariaDB / MySQL databases
```

Comment:
mysql is now the only supported db option

Reply:
Awesome! Ill remove any references to PostgreSQL and SQLite